docs/features/techdocs/concepts.md

```diff
@@ -8,13 +8,41 @@ Spotify's docs-like-code solution in Backstage
 This page describes concepts that are introduced with Spotify's docs-like-code
 solution in Backstage.
 
-### TechDocs Core Plugin
+### TechDocs Preparer
 
-The TechDocs Core Plugin is an [MkDocs](https://www.mkdocs.org/) plugin created
-as a wrapper around multiple MkDocs plugins and Python Markdown extensions to
-standardize the configuration of MkDocs used for TechDocs.
+Prepare is the first step of generating documentation for an entity. It fetches
+the source markdown files from the source code hosting provider (GitHub, GitLab,
+etc.) and passes the files to the generator for next steps.
 
-[TechDocs Core](https://github.com/backstage/mkdocs-techdocs-core)
+There are two kinds of preparers available -
+
+1. Common Git Preparer - Uses `git clone` on any repository url.
+2. Url Reader - Uses source code hosting provider's API to download files.
+   (Faster and recommended)
+
+## TechDocs Generator
```

Comment:
May want to check the headers across this file?  I would have expected preparer, generator, and publisher to be on a common heading size...

![image](https://user-images.githubusercontent.com/33203301/102257787-2d145800-3edb-11eb-8fe2-2dba243ce0e2.png)

Reply:
Ooops, great catch!